### DISCUSSION TITLE
paveloom-f fork of LBRY client and updates OK?

### DISCUSSION THREAD

**Discussion by yeehi** [paveloom-f/lbry-desktop#9] [Q&A] [answered]
1) If you install this fork of the LBRY client, and you are subsequently notified that updates are available, will the "updates" prevent the client from seeing masked content?   
  
2) So that we can try to verify that this fork works as intended, can you provide a link to an example of some masked content which:  
 
a) is masked by LBRY Inc. on Odysee 
b) is still seeded and visible using this forked client? 

3) What is the name for this forked client? the paveloom-f fork?

**Comment by paveloom** [comment, marked as answer]
Hello, @yeehi.

> 1. If you install this fork of the LBRY client, and you are subsequently notified that updates are available, will the "updates" prevent the client from seeing masked content?

Yes, self-updating the client will reset the [`lbryinc`](https://github.com/lbryio/lbryinc) module, and the blacklist will become functional again. When a new update is available, one can download the patched version from here and install it over the current installation.

> 2. So that we can try to verify that this fork works as intended, can you provide a link to an example of some masked content which:
> a) is masked by LBRY Inc. on Odysee
> b) is still seeded and visible using this forked client?

Usually, any recent DMCA takedown can suffice. The takedown notices are stored [here](https://github.com/lbryio/dmca), so one can check any of these. Since they're not backed up by LBRY, Inc. anymore, these tend to die out pretty quick, since the upload authors leave them behind.

Here is an example of a still hosted one (at the moment of writing): 
[odysee.com/@BootyControl:3/Sanya-Twerks-Like-That-Booty-Isolation-Twerk:9](https://odysee.com/@BootyControl:3/Sanya-Twerks-Like-That-Booty-Isolation-Twerk:9).

The related notice is [here](https://github.com/lbryio/dmca/blob/master/2021/2021-04-27-WorthIT-Solutions.md).

<details>
<summary>Proof</summary>
  <br>
  <img alt="Proof" src="https://user-images.githubusercontent.com/49961859/117483830-ed740880-af6e-11eb-9901-f62777f765e4.png")>
</details>

> 3. What is the name for this forked client? the paveloom-f fork?

Well, it isn't supposed to have a name (technically, the name is `paveloom-f/lbry-desktop`). It's just a fork that changes a few lines of code, so I don't think it's necessary to have a distinct name for that.

**Comment by paveloom** [comment]
Just a heads-up: since [`v0.51.1`](https://github.com/paveloom-f/lbry-desktop/releases/tag/v0.51.1), self-updating the client will not reset the blacklist anymore.
